### DISCUSSION TITLE
apicurio schmea registry - helm chart

### DISCUSSION THREAD

**Discussion by Woj-Teh** [Apicurio/apicurio-registry#5668] [Q&A]
Hello.
Does apicurio provide an official helm chart for installation schema registry? There is one available in artifact-hub, [apicurio-schema](https://artifacthub.io/packages/helm/apicurio-registry-helm/apicurio-registry), but from what I see it is not official and unfortunately not up to date either.
Thanks!

**Comment by Ganesh-403** [comment]
@Woj-Teh At the moment, **I don't believe Apicurio provides an official Helm chart**.

I checked the current documentation and getting started guides, and they document Docker/Podman images and Kubernetes deployment options, but there isn't an official Helm chart published or referenced by the Apicurio project itself. :contentReference[oaicite:0]{index=0}

The chart you found on Artifact Hub (`apicurio-registry-helm`) is maintained by the community (currently by **Ievgenii Shepeliuk**), not by the Apicurio organization. It is an actively maintained chart and supports deploying Apicurio Registry from an OCI registry, but it is not an official project deliverable. :contentReference[oaicite:1]{index=1}

So today your options are essentially:

- Use the community Helm chart if it meets your requirements.
- Deploy Apicurio Registry using Kubernetes manifests or your own Helm chart.
- Package the official container images (`quay.io/apicurio/apicurio-registry`) into an internal Helm chart if you need tighter control over upgrades and configuration. 

If having an official chart is important for your organization, it may be worth opening a feature request or starting a discussion in the project. An official chart maintained alongside the Registry releases would make version compatibility and upgrades much clearer.

If this solves your problem, feel free to mark it as the accepted answer so others can find it easily.
